premake5.lua

```diff
@@ -327,6 +327,10 @@ workspace("xenia")
     include("src/xenia/hid/sdl")
   end
 
+  if not os.istarget("windows") then
+    include("src/xenia/hid/keyboard")
```

Comment:
Maybe it's reasonable to rename it to keyboard_linux to prevent situation when people on windows try to use it